### DISCUSSION TITLE
Changing title bar color in a UWP react native App

### DISCUSSION THREAD

**Discussion by alphakex2** [microsoft/react-native-windows#9501] [Q&A] [answered]
How do I go about changing the title bar color? And I noticed that the generated code is C++ specific; how do I change it to C#?

**Comment by jonthysell** [comment, marked as answer]
If you want to generate a C# project, you should use the `--language cs` flag when running `react-native-windows-init`: https://microsoft.github.io/react-native-windows/init-cli

Also, to change the title bar color: https://docs.microsoft.com/en-us/windows/apps/design/shell/title-bar
